src/pages/api/profile/addresses/index.ts

```diff
@@ -104,8 +107,8 @@ const postHandler: Handler = async (request, response) => {
         true,
       );
 
-      response.statusCode = HttpStatus.OK;
-      response.json({ createdAddress });
+      response.statusCode = HttpStatus.CREATED;
```

Comment:
Changing `OK` to `CREATED` allows us to know if its an update or a create and avoid us to use `"updatedUserAddress" in parsedResponse` condition.